GOVERNANCE-stage.md

```diff
@@ -0,0 +1,135 @@
+
+The current list of teams is under the [teams](teams/README.md) directory.
+
+#### Reviewers
+
+Reviewers are individuals who actively make contribution and are willing to participate in the code review of new contributions. We identify reviewers from active contributors. The committers should explicitly solicit reviews from reviewers. High-quality code reviews prevent technical debt for long-term and are crucial to the success of the project. A pull request to the project has to be reviewed by at least one reviewer in order to be merged.
+
+Review access is by invitation only and must be approved by consensus of the maintainers. A reviewer is considered emeritus by their own declaration. An emeritus reviewer may request reinstatement of review access from the maintainers, which will be sufficient to restore him or her to active reviewer status.
+
+Review access can be revoked by consensus by the maintainers (except the reviewer in question if they are also a maintainer).
+
+#### Committers
+
+Committers are individuals who are granted the write access to the repositories belong to the team. A committer is usually responsible for a certain area or several areas of the code where they oversee the code review process. The area of contribution can take all forms, including code contributions and code reviews, documents, education, and outreach. Committers are essential for a high quality and healthy project.
+
+Commit access is by invitation only and must be approved by consensus of the maintainers. A committer is considered emeritus by their own declaration. An emeritus committer may request reinstatement of commit access from the maintainers, which will be sufficient to restore him or her to active committer status.
+
+Commit access can be revoked by consensus by the maintainers (except the committer in question if they are also a maintainer).
+
+#### Maintainers
+
+The maintainers consist group of active committers that moderate the discussion, manage the project release, and proposes new committers or maintainers. Potential candidates are usually proposed via an internal discussion among maintainers, followed by a consensus approval, i.e. a concrete number of approvals, and no vetoes. Any veto must be accompanied by reasoning. Maintainers should serve the community by upholding the community practices and guidelines TiDB a better community for everyone. Maintainers  should nominate new reviewer, committers and maintainers, and should also strive to only nominate new candidates outside of their own organization.
+
+Membership of the maintainers is by invitation only and must be approved by a consensus of the maintainers. A maintainer is considered emeritus by their own declaration. An emeritus member may request reinstatement to the maintainers, which will be sufficient to restore him or her to active maintainers.
+
+Membership of the maintainers can be revoked by a consensus vote of all the maintainers other than the member in question.
```

Comment:
What if there is only one maintainer? I think it should be able to remove a maintainer by active committee members.

Reply:
You are right the the committee can dominate the teams if necessary though we'd better avoid actually doing that. Let me try to explicitly describe this case.

Reply:
Updated 1c2668a1928435477ff28512627ac4a4a55a6fee . Though the better way I think is grow the maintainer group at the case. Because diversity is important of a community and in community we trust instead of setting up policies.